src/API/WesternStatesWater.WestDaat.Database/EntityFramework/Organization.cs

```diff
@@ -11,5 +11,7 @@ public Organization()
 
     public string Name { get; set; }
 
+    public string EmailDomain { get; set; }
```

Comment:
```suggestion
    public string EmailDomain { get; set; } = null!;
```

Update `Name` as well

Reply:
Update `src/API/WesternStatesWater.WestDaat.Database/EntityFramework/User.cs` as well

Reply:
And all other files it mentions in the warnings...